orchestration/dagster_orchestration/hca_orchestration/environments/stage_data_local_e2e.yaml

```diff
@@ -0,0 +1,15 @@
+resources:
+  beam_runner:
+    config:
+      target_class: "hca-transformation-pipeline"
+      working_dir: "../../"
+solids:
+  clear_staging_dir:
+    config:
+      staging_bucket_name: broad-dsp-monster-hca-dev-temp-storage
+      # staging_prefix_name: DETERMINED AT RUNTIME
```

Comment:
We need to create the staging_prefix at runtime so I'm excluding it here and then putting it in the dict we're passing to the pipeline running machinery at test-time.

@ThePletch you've been working in depth w/dagster configuration patterns so I'm interested in your thoughts on this one.

Reply:
If this works, it seems like an okay solution; is this something we'd want to generate during runtime for real runs? If so, we might not want it to be a part of the config. If this is just for testing, then I think it's okay